src/Xamarin.Android.Build.Tasks/Utilities/ResourceDesignerImportGenerator.cs

```diff
@@ -54,6 +71,20 @@ void CreateImportFor (bool isNestedSrc, IEnumerable<TypeDefinition> types, CodeM
 				foreach (var field in type.Fields) {
 					var dstField = new CodeFieldReferenceExpression (dstClassRef, field.Name);
 					var srcField = new CodeFieldReferenceExpression (srcClassRef, field.Name);
+					var fieldName = CreateIdentifier (type.Name, field.Name);
+					if (!resourceFields.Contains (fieldName)) {
+						Log.LogWarning (subcategory: null,
+							warningCode: "XA0106",
+							helpKeyword: null,
+							file: null,
+							lineNumber: 0,
+							columnNumber: 0,
+							endLineNumber: 0,
+							endColumnNumber: 0,
+							message: $"Skipping {fieldName}. Check your Nuget Package versions are compatible."
```

Comment:
Verbage: "Check **that** your NuGet package versions are compatible."

I'd also suggest using "Please check ...". No harm in being polite. :-)